modules/auth/bcryptUtils.js

```diff
@@ -1,19 +1,26 @@
 const bcrypt = require("bcrypt")
```

Comment:
I leave this comment here but you can use bcrypt on your service, it's not necessary to split it
it can leave to tech complexity as now you pass a param to existPassword